concepticondata/conceptlists/Greenhill-2015-2525.tsv

```diff
@@ -475,7 +475,7 @@ Greenhill-2015-2525-996	996	uncle	1327	UNCLE
 Greenhill-2015-2525-997	997	nephew	1063	NEPHEW
 Greenhill-2015-2525-998	998	rafter	829	RAFTER
 Greenhill-2015-2525-999	999	batten		
-Greenhill-2015-2525-1000	1000	youth		
+Greenhill-2015-2525-1000	1000	youth	1304	CHILD (YOUNG HUMAN)
```

Comment:
Here, I'd wonder if it is refering to "the youth" or "a youth". Can we tell this from the gloss alone?

Reply:
Hmm.. don't think I can tell here -- it's got 3 attestations, one from an old SIL list, one from a 18th century Russian scholar (https://en.wikipedia.org/wiki/Nicholas_Miklouho-Maclay),  and a sociolinguistic survey. I'll leave it unmapped.